test/notifications/display_test.dart

```diff
@@ -955,6 +963,74 @@ void main() {
       receiveFcmMessage(async, removeFcmMessage([message2], account: account2));
       check(testBinding.androidNotificationHost.activeNotifications).isEmpty();
     })));
+
+    test('removeNotificationsForAccount: removes notifications', () => runWithHttpClient(() => awaitFakeAsync((async) async {
+      await init();
+      final message = eg.dmMessage(from: eg.otherUser, to: [eg.selfUser]);
+
+      receiveFcmMessage(async, messageFcmMessage(message));
+      check(testBinding.androidNotificationHost.activeNotifications).isNotEmpty();
```

Comment:
nit: these are closely tied, so group together in one stanza:
```suggestion
      final message = eg.dmMessage(from: eg.otherUser, to: [eg.selfUser]);
      receiveFcmMessage(async, messageFcmMessage(message));
      check(testBinding.androidNotificationHost.activeNotifications).isNotEmpty();
```

(e.g., that `message` local exists only to be used on that next line, so those two lines belong in the same stanza)